Stripe/STPPaymentMethod+Private.h

```diff
@@ -10,7 +10,7 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface STPPaymentMethod ()
+@interface STPPaymentMethod (Private)
```

Comment:
Per my reply to @csabol-stripe's comment, I think this is a mistake. You're opting out of a compiler warning if methods declared in this header disappear/change names.